db/migrate/20200121220431_add_uniqueness_constraint_to_account_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniquenessConstraintToAccountUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_index :account_users, [:account_id, :user_id], unique: true, name: 'uniq_user_id_per_account_id'
```

Comment:
Do we really need this unique constraint? Later we might be having multiple roles for a user. I'm okay with this being kept here for now and removed when we add that or we can keep this by adding the role also to the index constraint.

What do you think?

Reply:
Kept it here since we haven't completely switched over from the single account per user model yet. I guess let's remove the db constraint for now and validate it with the model validation.